test/test_ops.py

```diff
@@ -1664,7 +1696,15 @@ class TestRefsOpsInfo(TestCase):
     def test_refs_are_in_python_ref_db(self, op):
         if op in self.skip_ref_ops:
             raise unittest.SkipTest(f"{op} does not have an entry in python_ref_db")
-        self.assertIn(op, self.ref_db_names)
+        # skip aliases since we don't want them defined in the ref db
+        if op in self.ref_alias_names_set:
+            return
+        self.assertIn(op, self.ref_db_names_set)
+
+    @parametrize("op", ref_db_names)
+    def test_ref_db_has_no_aliases(self, op):
+        self.assertTrue(op not in self.ref_alias_names_set,
+                        f"alias {op} in python_ref_db, which is not allowed")
```

Comment:
Can you improve this message?

Reply:
could you suggest one? i had tried some before i ended up on this one